src/translate_c.cpp

```diff
@@ -695,6 +684,40 @@ static bool qual_type_child_is_fn_proto(ZigClangQualType qt) {
     return false;
 }
 
+unsigned qual_type_get_ptr_alignment(Context *c, ZigClangQualType qt) {
+    const ZigClangType *ty = qual_type_canon(qt);
+    assert(ZigClangType_getTypeClass(ty) == ZigClangType_Pointer);
+    // Get the type it points to
+    const clang::PointerType *ptr_ty = reinterpret_cast<const clang::PointerType*>(ty);
+    ZigClangQualType base_qt = bitcast(ptr_ty->getPointeeType());
+    // Convert from bits to bytes
+    return ZigClangASTContext_getTypeAlign(c->ctx, base_qt) / 8;
```

Comment:
is the type alignment guaranteed to be multiple of 8? Could integer div interfere with ptr alignment comparison?